examples/dataset/keypoint_detection/keypoint_detection/upload_results.py

```diff
@@ -13,21 +13,36 @@
 # limitations under the License.
 from argparse import ArgumentParser
 from argparse import Namespace
+from typing import Any
+from typing import List
+from typing import Tuple
 
 import pandas as pd
 from keypoint_detection.metrics import compute_metrics
+from keypoint_detection.model import infer_from_df
 from keypoint_detection.model import infer_random
-from keypoint_detection.model import infer_retinaface
 from tqdm import tqdm
 
 import kolena
 from kolena.dataset import fetch_dataset
 from kolena.dataset import test
+from kolena.io import dataframe_from_csv
+from kolena.workflow.annotation import BoundingBox
+from kolena.workflow.annotation import Keypoints
```

Comment:
We should import these from `kolena.annotation` -- it's the longer term home for these post v1 (and we don't want to reference anything in `kolena.workflow` for datasets).